packages/cli/src/ceramic-cli-utils.ts

```diff
@@ -137,13 +137,6 @@ export class CeramicCliUtils {
      */
     static async change(docId: string, content: string, controllers: string, schemaDocId?: string): Promise<void> {
         const id = DocID.fromString(docId)
-
-        const commit = id.commit
-        if (commit) {
-            console.error(`No commits allowed. Invalid docId: ${id.toString()}`)
```

Comment:
This is a slight behavior change since `DocID.fromString()` does not currently error if you give it a string that represents a CommitID.  So where currently we'd error if you passed a CommitId string to `ceramic change` on the CLI, now we'll allow it and operate on the base DocId.  I'm not sure that's a change we actually want?  I kind of feel like `DocID.fromString()` should error if given a CommitID

Reply:
Makes sense.